scripts/localnet/localnet-cloud.sh

```
@@ -0,0 +1,645 @@
+start_docker_daemon() {
+  local dockerd_pid=""
+
+  if docker_ready; then
+    configure_docker_socket_permissions
+    return
+  fi
+
+  ensure_legacy_iptables
+
+  log "Starting Docker daemon with vfs storage driver..."
+  sudo nohup dockerd --host=unix:///var/run/docker.sock --pidfile="${DOCKERD_PID_FILE}" --storage-driver=vfs >"${DOCKERD_LOG_FILE}" 2>&1 &
```

Shellcheck (0.11.0)

[warning] sudo doesn't affect redirects. Use ..| sudo tee file

(SC2024)